src/main/java/com/fabriccommunity/spookytime/world/biome/SpookyBiomeFeatures.java

```diff
@@ -24,7 +25,11 @@
 	public static final Feature PUMPKIN = register("pumpkin", new DeceasedWildCropFeature(DefaultFeatureConfig::deserialize, Blocks.PUMPKIN.getDefaultState()));
 	
 	public static void addSpookyForestTrees(Biome biome) {
-	
+		if (biome instanceof SpookyForestBiome)
```

Comment:
I don't think we need to check the biome being passed in. This would only be called by a biome who wants the trees, so it wouldn't make sense to not add them.
If we _do_ keep the check, there should be braces. I'm not sure if the intention is to only check for the first feature, or all 3.

Reply:
I would have to agree. This check is not necessary at all.

Reply:
@Draylar its called by every biome rn. Ive been told lowlands is supposed to be like plains, not forest